pyxem/utils/correlation_utils.py

```diff
@@ -114,5 +114,41 @@ def _power(z, axis=0, mask=None, wrap=True, normalize=True):
         ).real
 
 
+def _pearson_correlation(z, mask=None):
+    """
+    Calculate Pearson cross-correlation of the image with itself
+    after rotation as a function of rotation
+
+     Parameters
+    ----------
+    z: np.array
+        Input image in 2D array
+    mask: np.array
+        A boolean mask to be applied.
+
+    Returns
+    -------
+    a: np.array
+        Pearson correlation of the input image
+
+    """
+    if mask is not None:
+        m = np.array(mask, dtype=bool)
+        mask_bool = ~m
+        mask_fft = np.fft.fft(mask_bool, axis=1)
+        n_unmasked = np.fft.ifft(mask_fft * mask_fft.conj()).real
+        n_unmasked[n_unmasked < 1] = 1
+        z[m] = 0
+        I_fft = np.divide(np.fft.fft(z, axis=1), n_unmasked)
```

Comment:
capital letters suggest classes in python, could this be written as `fft_intensity` instead?